src/components/Counter.js

```diff
@@ -19,11 +19,11 @@ class Counter extends Component {
         // should decrement or increment accordingly
         return (
             <p>
-                Clicked: {this.props.count} times
-                <button onClick={() => /* Fill me in */ }>
+                {/* Clicked: {this.props.count} times */}
+                <button onClick={this.increment }>
                     +
                 </button>
-                <button onClick={() => /* Fill me in */ }>
+                <button onClick={this.decrement }>
```

Comment:
you actually need to create an anonymous function that invokes the increment function here